speech/README.md

```diff
@@ -22,9 +22,11 @@ Usage:
     Usage: ruby speech_samples.rb <command> [arguments]
 
     Commands:
-      recognize               <audio-file.raw>
-      async_recognize         <audio-file.raw>
-      async_recognize_results <operation name>
+    recognize           <filename> Detects speech in a local audio file.
```

Comment:
Remove Line 5 in this file. I can't select it using Github review tools. 

` [Sign up for the Alpha](https://services.google.com/fb/forms/speech-api-alpha/).`